modules/special-rules/pages/transport-x.adoc

```diff
@@ -1,2 +1,16 @@
 = Transport (X)
 
+{blank}[TODO: Add introductory description.]
+
+---
+
+Where your unit has the {transport} ability:
+
+* The {transport} unit can carry as many Infantry units as we state in brackets.
+For example, the Rhino has {transport-2} -- so you can embark up to two Infantry units in a Rhino.
```

Comment:
```suggestion
* The {transport} unit can carry as many Infantry units as we state in brackets.
For example, the Rhino has {transport-2} -- so you can embark up to two Infantry units in a Rhino, as we explain in the Transport rules.
```
This should resolve your point below, without need for an inline link or a note.

Sometimes an inline link is the best option, but in general a 'related information' link is perfectly adequate, especially if we call it out like this.

Reply:
I'll commit my suggestion above so that it aligns with similar changes I've just pushed for Hitch (X). We can always change them both later if necessary.